.github/ISSUE_TEMPLATE/lib_change.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Library change
+about: Fix or improve issues with built-in type definitions like `lib.dom.d.ts`, `lib.es6.d.ts`, etc.
+title: ''
+labels: ''
+assignees: ''
+---
+# lib Update Request
+
+<!--
+  Please fill in each section completely. Thank you!
+
+  Are you here for one of these commonly-requested lib changes?
+    * Object.keys - see https://stackoverflow.com/questions/55012174/
+    * Array methods - see https://github.com/microsoft/TypeScript/issues/36554
+    * parseInt, parseFloat, isFinite, isNan, etc. - see https://github.com/microsoft/TypeScript/issues/4002
+
+  The DOM lib is maintained elsewhere and you can skip a step by filing issues/PRs for the DOM at that repo.
+  See https://github.com/microsoft/TypeScript-DOM-lib-generator
+-->
+
+## Configuration Check
+
+<!--
+  If you're missing common new methods like Array.includes, you have a misconfigured project.
+  Try setting lib: "es2020" and checking whether the type you want is present.
+
+  Conversely, if you are seeing built-in methods you expect to *not* see, check your 'lib' setting
+  or review your dependencies for lib/reference directives that might be polluting
+  your global scope. This is common when using the 'node' type library; see https://github.com/microsoft/TypeScript/issues/40184

```

Comment:
```suggestion
  your global scope. This is common when using the 'node' type library. See https://github.com/microsoft/TypeScript/issues/40184
```

Reply:
I know a separator you can use all the time when you want to make it seem like you're a better writer than you are; it's called semicolon.